semantic-conventions/src/tests/data/yaml/metrics.yaml

```diff
@@ -0,0 +1,22 @@
+groups:
+  - id: metric.foo
+    prefix: foo
+    type: metric
+    brief: "This document defines foo."
+    note: >
+        Details about foo.
+
+  - id: metric.foo.size
+    prefix: foo
+    type: metric
+    extends: metric.foo
+    metric_name: foo.size
```

Comment:
What is the `prefix` used for if `metric_name` is already the fully-qualified name? I'd either expect the prefix to not be used for metrics at all or that it would be prepended to the `metric_name` (which would then only be "size").

Reply:
`prefix` is used for the attribute names, not the metric names. 

I admit it is confusing from the current test. I will add in an attribute definition to the test so it's more clear how it works.

Reply:
Agreed it could be clearer 👍 I've gone ahead and made the `prefix: bar`.  See https://github.com/open-telemetry/build-tools/pull/79/commits/6a67c74f478f2dc70c07ae2c62183e78110c1e1e.